Makefile

```diff
@@ -1,13 +1,49 @@
 QUAY_USERNAME ?=
 QUAY_PASSWORD ?=
 
+IMAGE_REGISTRY ?= docker.io
+IMAGE_TAG ?= latest
+OPERATOR_IMAGE ?= rthallisey/hyperconverged-cluster-operator
+
 start:
 	./hack/deploy.sh
 
 clean:
 	./hack/clean.sh
 
+docker-build: docker-build-operator
+
+docker-build-operator:
+	docker build -f build/Dockerfile -t $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG) .
+
+docker-push: docker-push-operator
+
+docker-push-operator:
+	docker push $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG)
+
+cluster-up:
+	./cluster/up.sh
+
+cluster-down:
+	./cluster/down.sh
+
+cluster-sync:
+	./cluster/sync.sh
+
+cluster-clean:
+	CMD="./cluster/kubectl.sh" ./hack/clean.sh
+
 stageRegistry:
 	@REGISTRY_NAMESPACE=redhat-operators-stage ./hack/quay-registry.sh $(QUAY_USERNAME) $(QUAY_PASSWORD)
 
-.PHONY: start clean
+.PHONY: start \
```

Comment:
This is a nit, I don't care if you do anything about it, but the readability of the `.PHONY` isn't really important...maybe just put it all on one line?

Reply:
The convention is to separate each target in a different line.
Other projects in CNV does that too.